compiler/src/dmd/hdrgen.d

```diff
@@ -796,48 +796,34 @@ private void statementToBuffer(Statement s, ref OutBuffer buf, HdrGenState* hgs)
 
 private void dsymbolToBuffer(Dsymbol s, ref OutBuffer buf, HdrGenState* hgs)
 {
-    scope v = new DsymbolPrettyPrintVisitor(&buf, hgs);
-    s.accept(v);
+    toCBuffer(s, buf, *hgs);
 }
 
-private extern (C++) final class DsymbolPrettyPrintVisitor : Visitor
+void toCBuffer(Dsymbol s, ref OutBuffer buf, ref HdrGenState hgs)
```

Comment:
This function should be private or documented, but I see it already existed and you just moved it up.